src/components/home/HomePage.scss

```
@@ -0,0 +1,53 @@
+$black: #000000;
+.h-100vh {
+  height: 100vh;
+}
+
+.container {
+  .img-icon {
+    padding-bottom: 16px;
+  }
+  .h1-target {
+    font-family: "Open Sans", sans-serif;
+    font-style: normal;
+    font-weight: bold;
+    font-size: 20px;
+    line-height: 27px;
+    letter-spacing: 3px;
+    margin-bottom: 13px;
+  }
+  .h2-target {
+    font-family: "Open Sans", sans-serif;
+    font-style: normal;
+    font-weight: 500;
+    font-size: 18px;
+    line-height: 32px;
+  }
+  .p-target {
+    font-family: "Open Sans", sans-serif;
+    width: 321px;
+    height: 78px;
+    font-size: 14px;
+    line-height: 26px;
+    margin-bottom: 49px;
+  }
+  .home-link {
+    font-family: "Open Sans", sans-serif;
+    font-style: normal;
+    font-weight: bold;
+    text-transform: uppercase;
+    font-size: 12px;
+    line-height: 16px;
+    text-align: center;
+    letter-spacing: 2.4px;
+    color: $black;
+  }
+  .border-link:after {
+    display: block;
+    content: "";
+    padding-bottom: 11px;
+    border-bottom: 0.5px solid #000000;
```

border-bottom: 0.5px solid #000000;
```suggestion
    border-bottom: 0.5px solid $black;
```